android/build.gradle

```diff
@@ -108,11 +108,10 @@ dependencies {
   // For < 0.71, this will be from the local maven repo
   // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
   //noinspection GradleDynamicVersion
-  implementation "com.facebook.react:react-android:0.73.1"
+  implementation "com.facebook.react:react-native:+"
```

Comment:
This fixed the issue with test apps on `<=0.70`

Reply:
It causes kotlin files to appear broken while editing, unfortunately, because facebook/react imports appear broken. don't know how to fix that yet, but it compiles from the example app.